src/views/AutogenView.vue

```diff
@@ -45,11 +45,11 @@
                 <a-select
                   v-if="!dataView && $route.meta.filters && $route.meta.filters.length > 0"
                   :placeholder="$t('label.filterby')"
-                  :value="$route.query.filter || (['Admin', 'DomainAdmin'].includes($store.getters.userInfo.roletype) && $route.name === 'vm' ? 'all' : 'self')"
+                  :value="$route.query.filter || (['Admin', 'DomainAdmin'].includes($store.getters.userInfo.roletype) && ['vm', 'iso', 'template'].includes($route.name) ? 'all' : 'self')"
```

Comment:
@shwstppr It shows all but the api calls `templatefilter=self`. Could you fix that ?